### ISSUE TITLE
Missing argument for parameter 'trackAutomaticEvents' in call

### ISSUE THREAD

**Issue by brianstanley** [houseninjadojo/capacitor-mixpanel#32]
Hey everyone :)
I'm having some trouble when building the app in Xcode for iOS using Ionic and Capacitor.
<img width="810" alt="Captura de Pantalla 2022-11-01 a la(s) 12 36 14" src="https://user-images.githubusercontent.com/1711851/199273708-700e725f-a2fd-45ae-a810-7575747efbb0.png">
<img width="511" alt="Captura de Pantalla 2022-11-01 a la(s) 12 36 17" src="https://user-images.githubusercontent.com/1711851/199273719-f4d37f5a-72b8-4208-8c0b-6020dff2be8f.png">

This is my Podfile:
```
platform :ios, '13.0'
use_frameworks!

# workaround to avoid Xcode caching of Pods that requires
# Product -> Clean Build Folder after new Cordova plugins installed
# Requires CocoaPods 1.6 or newer
install! 'cocoapods', :disable_input_output_paths => true

def capacitor_pods
  pod 'Capacitor', :path => '../../node_modules/@capacitor/ios'
  pod 'CapacitorCordova', :path => '../../node_modules/@capacitor/ios'
  pod 'CapacitorCommunityFirebaseAnalytics', :path => '../../node_modules/@capacitor-community/firebase-analytics'
  pod 'CapacitorCommunityFirebaseCrashlytics', :path => '../../node_modules/@capacitor-community/firebase-crashlytics'
  pod 'CapacitorCommunityKeepAwake', :path => '../../node_modules/@capacitor-community/keep-awake'
  pod 'CapacitorApp', :path => '../../node_modules/@capacitor/app'
  pod 'CapacitorBrowser', :path => '../../node_modules/@capacitor/browser'
  pod 'CapacitorDevice', :path => '../../node_modules/@capacitor/device'
  pod 'CapacitorHaptics', :path => '../../node_modules/@capacitor/haptics'
  pod 'CapacitorKeyboard', :path => '../../node_modules/@capacitor/keyboard'
  pod 'CapacitorShare', :path => '../../node_modules/@capacitor/share'
  pod 'CapacitorSplashScreen', :path => '../../node_modules/@capacitor/splash-screen'
  pod 'CapacitorStatusBar', :path => '../../node_modules/@capacitor/status-bar'
  pod 'HouseninjaCapacitorMixpanel', :path => '../../node_modules/@houseninja/capacitor-mixpanel'
  pod 'CordovaPlugins', :path => '../capacitor-cordova-ios-plugins'
end

target 'uugot.it' do
  capacitor_pods
  # Add your Pods here
end
```
Any thoughts? I'm kind of new around here. If you need further information let me know and I'll be happy to share with you!

Thanks

**Comment by sSarahlei** [comment]
I am also having this issue
This is blocking my app release.

**Comment by brianstanley** [comment]
As a workaround I directly edited that line with Xcode and added it the trackAutomaticEvents argument but of course this is not the way...

**Comment by mileszim** [comment]
fixed in #40
